path_rotate_root.go

```diff
@@ -0,0 +1,256 @@
+package azuresecrets
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/vault-plugin-secrets-azure/api"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/helper/parseutil"
+	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/mitchellh/mapstructure"
+)
+
+func pathRotateRoot(b *azureSecretBackend) *framework.Path {
+	return &framework.Path{
+		Pattern: "rotate-root",
+		Fields: map[string]*framework.FieldSchema{
+			"expiration": {
+				Type: framework.TypeString,
+				// 28 weeks (~6 months) -> days -> hours
+				Default:     (28 * 7 * 24 * time.Hour).String(),
```

Comment:
How was the default value of 6 months chosen here? Would it be good to go with [Azure's default value](https://docs.microsoft.com/en-us/graph/api/application-addpassword?view=graph-rest-1.0&tabs=http#request-body) in here?

Reply:
It's the default when creating a credential in the Azure UI. I have no idea why it's different from the API's default.